pychunkedgraph/backend/chunkedgraph_comp.py

```diff
@@ -67,3 +98,47 @@ def get_latest_roots(cg,
         root_ids.extend(result)
 
     return np.array(root_ids, dtype=np.uint64)
+
+def get_changed_roots(cg,
+                      time_stamp_start: datetime.datetime,
+                      time_stamp_end: Optional[datetime.datetime] = None,
+                      n_threads: int = 1) -> Sequence[np.uint64]:
+
+    # Create filters: time and id range
+    max_seg_id = cg.get_max_seg_id(cg.root_chunk_id) + 1
+
+    n_blocks = np.min([n_threads * 3 + 1, max_seg_id])
+    seg_id_blocks = np.linspace(1, max_seg_id, n_blocks, dtype=np.uint64)
```

Comment:
that certainly is an overkill for most timestamps
but its not easy to find a good range because I do not guarantee monotonicity of root ids in time

Reply:
added a feature to optionally specify a min_seg_id